pkgs/servers/code-server/default.nix

```diff
@@ -43,12 +43,43 @@ in stdenv.mkDerivation rec {
     '';
   };
 
+  esbuild-code-server-build = buildGoModule rec {
```

Comment:
Can#t we use the nixpkgs one?

Reply:
👍 Updated this to use the nixpkgs one and patch the JS client with the version of the nixpkgs esbuild.